src/components/AddPet.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import PetForm from "./PetForm";
+import { FORM_ERROR } from "final-form";
+import { addPet } from "../service/petsAPI";
+import { useNavigate } from "react-router-dom";
+
+export default function EditPet() {
+  const initialValues = {
```

Comment:
I suggest using `useMemo` or `useRef` to store the value for `initialData` as the fact that the initial values object is recreate on every re-render of this component might cause the form to have unexpected behaviour. As a best practice it is best to have props sent to other components not change unless necessary